flink-streaming-connectors/flink-connector-kafka-base/src/test/java/org/apache/flink/streaming/connectors/kafka/testutils/MockRuntimeContext.java

```diff
@@ -71,8 +71,7 @@ public MockRuntimeContext(
 		ExecutionConfig execConfig,
 		Object checkpointLock) {
 
-		this(numberOfParallelSubtasks, indexOfThisSubtask, execConfig, checkpointLock,
-			DefaultTimeServiceProvider.create(Executors.newSingleThreadScheduledExecutor()));
+		this(numberOfParallelSubtasks, indexOfThisSubtask, execConfig, checkpointLock, null);
```

Comment:
Instead of passing `null` you can directly insert this line from below: `DefaultTimeServiceProvider.createForTesting(Executors.newSingleThreadScheduledExecutor(), this.checkpointLock)` and avoid the null check below.